File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/PYTHON/6_Stack.py
from  collections import deque
if __name__ == '__main__':
    #Tao 1 stack kieu int
    myStack = deque()

    #2. add cac value vao stack moi tao
    myStack.append(1)
    myStack.append(2)
    myStack.append(3)

    print("myStack:", end=" ")
    while len(myStack)>0:
        print(myStack[-1], end=" ") #ktra p.tu cuoi cua stack la value nao
        myStack.pop() #loai bo p.tu cuoi

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/6_Stack.js
import java.util.Stack;
public class StackInJava {
    public static void main(String[] args) {
        //Tao 1 stack kieu int
        Stack<Integer> myStack = new Stack<>();

        //2. add cac value vao stack moi tao
        myStack.add(1);
        myStack.add(2);
        myStack.add(3);

        System.out.print("myStack: ");
        while(!myStack.isEmpty()){
            System.out.print(myStack.peek()+ " "); //ktra p.tu cuoi cua stack la value nao
            myStack.pop(); //loai bo p.tu cuoi
        }

    }
}
